resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,10973 @@
+        <pre class="code--block">
+          {
+            "code": "BAD_REQUEST",
+            "message": "Bad Request",
+            "errors": [
+              {
+                "code": "INVALID_FORMAT",
+                "message": "Invalid format has been used",
+                "path": "/accountId"
+              }
+            ]
+          }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td><p>Request for an account that is open or active</p><p class="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a></p></td>
+                    <td>
+        <pre class="code--block">
+        {
+          "accountId": "2000000403"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">403 (Forbidden)</code></p>
+        <pre class="code--block">
+        {
+          "code": "INVESTOR_ACCOUNT_ALREADY_OPEN",
+          "message": "The account already has a status of Open"
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td><p>Request for an account that is closed with a closure reason as transferred out</p><p class="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a></p></td>
+                    <td>
+        <pre class="code--block">
+        {
+          "accountId": "0000000403"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">403 (Forbidden)</code></p>
+        <pre class="code--block">
+        {
+          "code": "INVESTOR_ACCOUNT_ALREADY_CLOSED",
+          "message": "You cannot reinstate this account because it was closed with a closure reason of transferred out"
+        }
+        </pre>
+                    </td>
+                </tr>
+                 <tr>
+                    <td><p>Request for an account that is closed with a closure reason as cancelled</p><p class="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a></p></td>
+                    <td>
+        <pre class="code--block">
+        {
+          "accountId": "1000000403"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">403 (Forbidden)</code></p>
+        <pre class="code--block">
+        {
+          "code": "INVESTOR_ACCOUNT_ALREADY_CANCELLED",
+          "message": "You cannot reinstate this account because it was closed with a closure reason of cancellation"
+        }
+        </pre>
+                    </td>
+                </tr>
+                 <tr>
+                    <td><p>Request for an account that is closed with a closure reason as cancelled</p><p class="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a></p></td>
```

Comment:
This is one of the things I found incorrect in this YAML which was transferred from the RAML. This should be when an investor has failed a compliance check. However, can ignore for now but please update link from `#section/Testing` to `/api-documentation/docs/api/service/lisa-api/2.0#testing` once the duplicated Testing section in this file has been removed. Thanks